tools/testbench/ll_schedule.c

```diff
@@ -12,10 +12,6 @@ int schedule_task_init_ll(struct task *task, uint32_t uid, uint16_t type,
 			  enum task_state (*run)(void *data), void *data,
 			  uint16_t core, uint32_t flags)
 {
-	int ret;
-
-	ret = schedule_task_init(task, uid, type, priority, run, data, core,
-				 flags);
-
-	return ret;
+	return schedule_task_init(task, uid, type, priority, run, data, core,
+				  flags);
```

Comment:
Didn't I just merge this patch early today?

Reply:
It is not showing me as merged.
Also when I created a new branch and pushed it the old two commits are always in the front of the list. 
How to only send new branch's commits as a single pull request?
I used "git push origin cleanup" where cleanup is one of the branches in local repo with 4 commits and Master branch has 2 commits which I had sent earlier.
Sorry for the inconvenience.
I am new github.
Thanking Payal:)

Reply:
@PayalKshirsagar you have a rebase your local branch on top of the latest SOF master.

Reply:
Sorry, I didn't get it:(

Reply:
@PayalKshirsagar each time you create a new patch OR a new patch series, your current working branch must match the remote branch commit against you are adding the commits.

The main branch of SOF repo is named master.

So, lets say you are working on 2 different patches. Patch1 and Patch2. If those patches are not related please add them in separate Pull Requests.
- Pull request 1 - > with patch 1 (for this you create your local branch named branch1)
- Pull request 2 -> with patch 2 (for this you create your local branch named branch2)

When PR1 gets merged patch1 is merged to master. Now if you do modifications to PR2 make sure you also rebase your branch2 so that it contains changes already accepated.

This can be done using (assuming you are on branch2)
* git fetch origin
* git rebase origin/master

Reply:
Okay, Thank you:)